hermes-engine.podspec

```diff
@@ -20,13 +20,13 @@ Pod::Spec.new do |spec|
   # The podspec would be serialized to JSON and people will download prebuilt binaries instead of the source.
   # TODO(use the hash field as a validation mechanism when the process is stable)
   spec.source      = ENV['hermes-artifact-url'] ? { http: ENV['hermes-artifact-url'] } : { git: "https://github.com/facebook/hermes.git", tag: "v#{spec.version}" }
-  spec.platforms   = { :osx => "10.13", :ios => "10.0" }
+  spec.platforms   = { :osx => "11.0", :ios => "10.0" }
```

Comment:
Maybe because of this version bump?

Reply:
I reverted that bump, I don't think that it's essential for Catalyst support.
Now CircleCI fails on Android tests, but my commits have nothing to do with the android part of code.

Reply:
Yeah do not worry about the Android tests. The android emulator image is somehow missing.